rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml

```diff
@@ -146,3 +146,42 @@ spec:
             team: pipelines
             # link will be available once https://issues.redhat.com/browse/SRVKP-5899 is complete
             runbook_url: https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/slos/chains-performance.md
+        - alert: ResultsDeadlock
+          expr: |
+            sum by (source_cluster) (increase(tekton_pipelines_controller_pipelinerun_count[5m])) > 0
+            and
+            sum by (source_cluster) (increase(grpc_server_handled_total{job='tekton-results-api', grpc_service=~'tekton.results.v1alpha2.*'}[5m])) == 0
```

Comment:
How critical is it to filter by the `v1alpha2` version in all these expressions? Could we use just `tekton.results.*` instead so this doesn't break unexpectedly in the event of a Tekton results version bump?

Reply:
it makes a difference ... unfortunately there is non-results api activity that occurs within the grpc server started within tekton-results-api that we should filter out

I just did a comparison query on prod-rh01 and these diffs showed up:
![grpc-with-without-filter](https://github.com/user-attachments/assets/f19c6bd4-1694-4a1a-bf5a-bc880cfd9bb7)

Reply:
wait ... maybe you meant just remove `v1alpha2` .... just a sec I'll try that

Reply:
yeah you did (I read too fast) ... they are the same when I query on prod-rh01 .... I'll update

Reply:
updated @amisstea sorry for the comment churn ;-)